src/storybook/pages/home/home.common.ts

```diff
@@ -1,5 +1,5 @@
 import type { Args } from '@storybook/web-components';
```

Comment:
I suggest to remove all the lean switches in this file (and not use it in lean story) (footer and navigation are for public websites mainly)